hack/tools/generate-yaml/Dockerfile

```diff
@@ -19,6 +19,9 @@ ARG BASE_IMAGE=gcr.io/cluster-api-provider-vsphere/extra/manifests-base:v0.4.2-b
 FROM ${BASE_IMAGE}
 LABEL "maintainer" "Andrew Kutz <akutz@vmware.com>"
 
+# install govc, used to check vSphere version
+RUN curl -L https://github.com/vmware/govmomi/releases/download/v0.21.0/govc_linux_386.gz | gunzip > /usr/local/bin/govc && chmod +x /usr/local/bin/govc
```

Comment:
Hi @andrewsykim,

Should this be `linux_amd64`, https://github.com/vmware/govmomi/releases/download/v0.21.0/govc_linux_amd64.gz?

Reply:
oops, copied the wrong URL, thanks